extensions/das/artifacts/metering-dashboard/capps/ppaas-metering-service/GadgetMemberCount/Member_Count/js/gadgetconf.js

```diff
@@ -16,7 +16,6 @@
 var gadgetConfig = {
     "id": "Member_Count",
     "title": "Member_Count",
-    "datasource": "MEMBER_COUNT",
```

Comment:
Yes. It was there earlier and I used this variable to define the table name for the gadget